attribute-lister.go

```diff
@@ -0,0 +1,7 @@
+package vugu
+
+// AttributeLister implements the required functionality, required by the `:=` attribute setter.
+// Types may implement this interface, to
+type VGAttributeLister interface {
+	ListAttributes() []VGAttribute
```

Comment:
Let's call the method `AttributeList` to go along with the convention of "interface `Aer` has method `A`".  (The "VG" prefix is a little funky - all of this VGNode and VGAttribute might move to a separate package at some point in order to get rid of that prefix, but I don't want to get into that now.)